ecdsa/Cargo.toml

```diff
@@ -46,8 +46,7 @@ hazmat = []
 pkcs8 = ["digest", "elliptic-curve/pkcs8", "der"]
 pem = ["elliptic-curve/pem", "pkcs8"]
 serde = ["elliptic-curve/serde", "pkcs8", "serdect"]
-signing = ["arithmetic", "digest", "hazmat", "rfc6979"]
-verifying = ["arithmetic", "digest", "hazmat"]
+signature = ["arithmetic", "digest", "hazmat", "rfc6979"]
```

Comment:
Feels possibly a bit overloaded with the `signature` crate, though I'm hard pressed to come up with something better than e.g. `algorithm`